woocommerce/google-listings-and-ads
# Implement RequestReview in the Merchant class

### Changes proposed in this Pull Request:

This PR implementers Get Account Review Status and Request a Review in the MErchant Class instead of doing it with WooCommerce Connect Server (via Middleware)

The reason of this is in order to avoid unnecessary manager quota requests

See slack context here: p1713778408402949-slack-C02BB3F30TG 


### Screenshots:


https://github.com/woocommerce/google-listings-and-ads/assets/5908855/a4d57af3-9d0b-411e-a394-8dbabcad80ac




### Detailed test instructions:


1. Checkout this PR
2. Connect MC with an account  disapproved and eligible for review request
3. See the disapproved message in the product feed tab
4. See the request review notice
5. Request a review successfully and see how the status goes to "Under review"
6. Check now an approved or pending to approve account
7. See the review status message is correct in product feed tab


### Additional details:


### Changelog entry

> Update - Implement Account Request Review Requests in the extension
